locale/de/app.po

```diff
@@ -0,0 +1,251 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Shouldn't everything in the same app.po file? And why is there an app.po app.pot and app.edit.pot?